packages/evm/src/interpreter.ts

```diff
@@ -374,6 +379,13 @@ export class Interpreter {
 
     let gas = opInfo.feeBigInt
 
+    // Cache pre-gas memory size if doing tracing (EIP-3155)
+    let memorySize = BIGINT_0
```

Comment:
Suggestion to rename this to something like `memorySizeCached` or something, this now looks like "current memory size" but it is not the purpose of this var. (It is to cache the memory before any eventual expansions of the memory in the gas calculation)